### DISCUSSION TITLE
define head in nuxt.config ?

### DISCUSSION THREAD

**Discussion by KareemDa** [nuxt/nuxt#18339] [Questions]
I'm reading [nuxt.config documentation](https://v3.nuxtjs.org/docs/directory-structure/nuxt.config)

But I've figured out that there is no `head` option. is it replaced with `meta`? [see link](https://v3.nuxtjs.org/docs/directory-structure/nuxt.config#meta)

you can notice that there is another `meta` object inside it. 

is that right ?  
```
meta: {
	meta: [
		{ charset: 'utf-8' },
		{ name: 'viewport', content: 'width=device-width, initial-scale=1' },
		{ hid: 'description', name: 'description', content: '' },
		{ name: 'format-detection', content: 'telephone=no' }
	],

	link: [{ rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }]
}
```

another question: where to add page title, htmlAttrs ?

**Comment by Zielgestalt** [comment]
You can add the title the same way it was before in the head option:

`
meta: {
    title: 'Your title',
    meta: [],
    link: [{ rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }]
  },
`

But htmlAttrs seems not to work anymore.
